homework/Thu_Pham/lesson_1/191-number-of-1-bits.js

```diff
@@ -0,0 +1,9 @@
+//Solution for https://leetcode.com/problems/number-of-1-bits/
+
+/*
+ * @param {number} n - a positive integer
+ * @return {number}
+ */
+ var hammingWeight = function(n) {
+    return n.toString(2).replace(/0/g, "").length;
```

Comment:
this approach will pass leetcode test, but in a real interview, I think interviewer will ask you to impl the n.toString(2) from scratch. How about using the n&(n-1) operation to solve this problem?